cmd/params.go

```diff
@@ -179,19 +205,46 @@ func params(cmd *cobra.Command, args []string) {
 		res := <-results
 		orderedResults[res.Index] = res
 	}
+
 	if !Json {
-		// will only know the proper header once we have a successful run
-		headerPrinted := false
+
+		paramSet := []string{}
+		for _, res := range orderedResults {
+			results := res.Result
+			paramSet = append(paramSet, results.ParameterNames...)
+		}
+
+		paramSet = removeDuplicateValues(paramSet)
+		for i, s := range paramSet {
+			paramSet[i] = strings.ReplaceAll(s, ",", "_")
+		}
+
+		fmt.Println("dir,error,termination," + strings.Join(paramSet, ","))
 		for _, res := range orderedResults {
+			s := make([]string, len(paramSet))
+			for i, _ := range s {
+				s[i] = ""
+			}
+
+			absolutePath := dir + "/" + modelDirs[res.Index]
```

Comment:
sidenote: I _do_ think we should keep the end of the path like this (essentially returning the path to the dir that the `.ext` is in, instead of the the path to the `.ext`) because that is what `bbr` thinks of as the "absolute model path" and is consistent with what the other `_log()` functions return.

Reply:
I can't reproduce the double "//".  If I change line 229 to: 
```
absolutePath := dir + modelDirs[res.Index]
```
and run: 
```
bbi nonmem params --dir ~/Desktop/bbr/inst/model/nonmem/basic/test_batch_params
```
I get directories that look like this:
 ```
/Users/AnnaN/Desktop/bbr/inst/model/nonmem/basic/test_batch_params1
```

I think it may have to do with whether or not you pass `/Desktop/bbr/inst/model/nonmem/basic/test_batch_params` or `/Desktop/bbr/inst/model/nonmem/basic/test_batch_params` as the `--dir`. To allow for either, this has been updated to: 

```
			var absolutePath string
			if strings.HasSuffix(dir, "/"){
				absolutePath = dir + modelDirs[res.Index]
			} else {
				absolutePath = dir + "/" + modelDirs[res.Index]
			}
```

in the latest commit.